spec/requests/api/moves_controller_role_access_rswag_spec.rb

```diff
@@ -44,7 +47,7 @@
 
       it 'returns only moves belonging to suppliers' do
         response_ids = response_json['data'].map { |move| move['id'] }.sort
-        data_ids = pentonville.moves_from.pluck(:id).sort
```

Comment:
Sorting by id here is worrying, tbh. We should really define standard orders for collections of our entities and optimise by ordered indexes 🤔 for the default sort order. Another PR. If we don't care about the order maybe we can use the `contain_exactly` matcher and remove the `sort` on line 47